hexa/plugins/connector_airflow/datagrids.py

```diff
@@ -39,24 +40,30 @@ class DAGRunGrid(Datagrid):
         text="get_label",
         icon="get_icon",
     )
-    favorite = BooleanColumn(value="is_favorite")
     execution_date = DateColumn(date="execution_date", label="Execution date")
-    user = TextColumn(text="user.display_name")
     state = StatusColumn(value="status")
     duration = DurationColumn(duration="duration", short_form=True)
+    user = TextColumn(text="user.display_name")
 
     view = LinkColumn(text="View")
 
     @staticmethod
     def get_label(run: DAGRun) -> str:
         if hasattr(run, "favorite") and getattr(run, "favorite") is not None:
             return getattr(run, "favorite")
-
-        return run.run_id
+        elif hasattr(run, "run_id") and re.match(r"^manua\w", getattr(run, "run_id")):
```

Comment:
Please use `run.run_id.startswith()` instead of `re.match`. Simpler is always better than complex ;)